src/CalcViewModel/ApplicationViewModel.cpp

```diff
@@ -257,7 +258,8 @@ task<void> ApplicationViewModel::HandleToggleAlwaysOnTop(float width, float heig
         IsAlwaysOnTop = success;
     }
     SetDisplayNormalAlwaysOnTopOption();
-}
+};
```

Comment:
No need for the `;`
```suggestion
}
```

Reply:
I believe this is needed for the compiler to recognize this as a function.